internal/bminventory/inventory.go

```diff
@@ -244,6 +246,37 @@ func (b *bareMetalInventory) updatePullSecret(pullSecret string, log logrus.Fiel
 	return pullSecret, nil
 }
 
+func mergeIgnitionConfig(base []byte, overrides []byte) (string, error) {
+	baseConfig, report, err := ign_3_1.Parse(base)
```

Comment:
u have validateIgnitionConfig why not to use it here?

Reply:
Because it doesn't return the config. I could change it if you think it would be better, or just remove the validate function entirely as it's only a call and two error checks and only used in one other place ... :man_shrugging: